src/snowflake/sqlalchemy/snowdialect.py

```diff
@@ -669,7 +672,10 @@ def get_columns(self, connection, table_name, schema=None, **kw):
         if schema_columns is None:
             # Too many results, fall back to only query about single table
             return self._get_table_columns(connection, table_name, schema, **kw)
-        return schema_columns[self.normalize_name(table_name)]
+        normalized_table_name = self.normalize_name(table_name)
+        if normalized_table_name not in schema_columns:
+            raise sa_exc.NoSuchTableError()
```

Comment:
we should document the changing behavior in the release notes that `NoSuchTableError` is raised instead of `KeyError`.

Reply:
I made a new minor version in the description markdown file with today's date. If there is a different way you want to handle this let me know, but I see you edited it manually last week so I followed your pattern.

Reply:
it looks awesome, thanks!